packages/web/components/Landing/OnboardingGame/metagame-onboarding-game.json

```diff
@@ -734,7 +734,7 @@
     "1357573e-5f79-420e-9e61-6da47f341546": {
       "theme": "moss",
       "title": null,
-      "content": "<p>There are a few ways to participate.</p><p>You can join as a player to take an active part in the creation of MetaGame.</p><p>You can join as a patron to contribute passively; simply water Seeds &amp; enjoy being a member.</p><p>Or you can join as a guild, offering your products &amp; services and receiving support from MetaGame.</p><p>- <a href=\"/profile/setup/name\" target=\"__blank\" rel=\"noopener noreferrer nofollow\" title=\"https://metagame.wtf/profile/setup/name\">Join as a player</a></p><p>- <a href=\"/join/patron\" target=\"__blank\" rel=\"noopener noreferrer nofollow\" title=\"https://metagame.wtf/join/patron\">Join as a patron</a></p><p>- <a href=\"/join/guild\" target=\"__blank\" rel=\"noopener noreferrer nofollow\" title=\"https://metagame.wtf/join/guild\">Join as a guild</a></p>",
+      "content": "<p>There are a few ways to participate.</p><p>You can join as a player to take an active part in the creation of MetaGame.</p><p>You can join as a patron to contribute passively; simply water Seeds &amp; enjoy being a member.</p><p>Or you can join as a guild, offering your products &amp; services and receiving support from MetaGame.</p><p>- <a href=\"/onboarding/guide\" target=\"__blank\" rel=\"noopener noreferrer nofollow\" title=\"https://metagame.wtf/onboarding/guide\">Join as a player</a></p><p>- <a href=\"/onboarding/guide\" target=\"__blank\" rel=\"noopener noreferrer nofollow\" title=\"https://metagame.wtf/onboarding/guide\">Join as a patron</a></p><p>- <a href=\"/join/guild\" target=\"__blank\" rel=\"noopener noreferrer nofollow\" title=\"https://metagame.wtf/join/guild\">Join as a guild</a></p>",
```

Comment:
```suggestion
      "content": `<p>There are a few ways to participate.</p><p>You can join as a player to take an active part in the creation of MetaGame.</p><p>You can join as a patron to contribute passively; simply water Seeds &amp; enjoy being a member.</p><p>Or you can join as a guild, offering your products &amp; services and receiving support from MetaGame.</p><ul><li><a href="/onboarding/guide" target="_blank" rel="noopener noreferrer nofollow">Join as a player</a></li><li><a href="/onboarding/guide" target="_blank" rel="noopener noreferrer nofollow">Join as a patron</a></li><li><a href="/join/guild" target="_blank" rel="noopener noreferrer nofollow">Join as a guild</a></li></ul>`,
```

Reply:
Adding this suggestion results in this error 
![image](https://user-images.githubusercontent.com/31413091/202763071-5924bd97-e63a-42a8-9242-b65731314f0a.png)
Seems the format isn't a valid json value